### PR TITLE
Legacy review Router multi-site sample

### PR DESCRIPTION
**Pull request by SzymonPobiega**

### CODE REVIEW

File: samples/router/sites/sample.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ include: bridge-to-router-note
 ## Running the project
 
  1. Start the projects in debug mode.
- 1. Press <kbr>Enter</kbr> a couple of times.
- 1. Observe the `PingHandler` logging processed message IDs in the Server window.
- 1. Observe the `PongHandler` logging processed message IDs in the Client window.
+ 1. Press <kbd>Enter</kbd> a couple of times.
```
Comment (PhilBastian) [inline samples/router/sites/sample.md:14]: "couple" is 2: is it exactly 2 times that the Enter key needs to be pressed?

File: samples/router/sites/sample.md [hunk 2]
```diff
@@ -44,22 +44,22 @@ It does not contain any forwarding configuration because the `Pong` reply messag
...
+The **Server** project contains an NServiceBus endpoint that processes the `Ping` messages and sends the `Pong` messages as a response. 
 
 
 ## How it works
 
-The Client sends the `Ping` message to the router using local transport. The message carries a special header that denotes which site or sites the message should be delivered to.
+The **Client** endpoint sends the `Ping` message to the router using local transport. The message carries a special header that denotes which site or sites the message should be delivered to.
 
-When the router in the origin site (RouterA) picks up the message it looks at the header and dispatches the message to the router in the destination site(s) using the tunnel transport (MSMQ in this sample). Before dispatching, the router stamps the message with a header denoting the origin site.
+When the router in the origin site (**RouterA**) picks up the message it looks at the header and dispatches the message to the router in the destination site(s) using the tunnel transport (MSMQ in this sample). Before dispatching, the router stamps the message with a header denoting the origin site.
```
Comment (PhilBastian) [inline samples/router/sites/sample.md:61]: Should the sample be upgraded away from MSMQ?

**Review by PhilBastian — APPROVED** [review]

**Comment by mikeminutillo** [comment]
@SzymonPobiega have you reviewed the whole showcase sample? Should we update the review date on the main article?

File: samples/router/sites/sample.md (hunk 1 above)
Comment (mikeminutillo) [reply samples/router/sites/sample.md:14]:
```suggestion
 1. Press <kbd>Enter</kbd> a few times.
```

**Review by mikeminutillo — APPROVED** [review]

**Pull request merged**